syntax = "proto3";
package dpi;

message RegistrateRequest {
    string ip = 1;
    uint32 port = 2;
    uint32 type = 3;
}

message UpdateNIDReq {
	repeated uint32 nodeid = 1;
	repeated string connection = 2;
	repeated uint32 type = 3;
}